vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java

```diff
@@ -51,9 +51,10 @@
 
 	private final HttpServletRequest request;
 
-	/** 
+	/**
 	 * @deprecated CDI eyes only
 	 */
+	@Deprecated
```

Comment:
no need for this one. we use only the javadoc comment by convention (;

Reply:
eclipse things